### PR TITLE
Added Left/Right menu offset properties for portrait and landscape

### PR DESCRIPTION
**Pull request by drumbart**

### CODE REVIEW

**Comment by aryaxt** [comment]
Awesome thanks a lot

Your branch seems to be out of date. Can you open this against Develop instead.
I'm also thinking maybe it makes sense to get rid of portraitSlideOffset and landscapeSlideOffset, It might be confusing having all those offset values.

File: SlideMenu/Source/SlideNavigationController.m [hunk 1]
```diff
@@ -38,6 +38,7 @@ @interface SlideNavigationController() <UIGestureRecognizerDelegate>
 @property (nonatomic, strong) UIPanGestureRecognizer *panRecognizer;
 @property (nonatomic, assign) CGPoint draggingPoint;
 @property (nonatomic, assign) Menu lastRevealedMenu;
+@property (nonatomic, assign) Menu currentMenu;
```
Comment (aryaxt) [inline SlideMenu/Source/SlideNavigationController.m]: Why not use lastRevealedMenu?

**Comment by drumbart** [comment]
Sure, I'll open it against Develop.
I wouldn't remove portraitSlideOffset and landscapeSlideOffset just yet, to preserve backwards compatibility. I'd suggest marking those as deprecated and remove in the next major version.

**Comment by drumbart** [comment]
As for the currentMenu, it's the naming that's clearer when working with it in the slide handling methods.
I also didn't want to run into any potential bugs with lastRevealedMenu. I'll have a look again and refactor when I got the chance.

**Pull request closed without merging**